tests/aws/services/secretsmanager/test_secretsmanager.py

```diff
@@ -263,11 +264,11 @@ def test_call_lists_secrets_multiple_times_snapshots(
 
     @markers.aws.validated
     def test_list_secrets_filtering(self, aws_client, create_secret):
-        unique_id = short_uid()
-        secret_name_1 = f"testing1/one-{unique_id}"
-        secret_name_2 = f"/testing2/two-{unique_id}"
-        secret_name_3 = f"testing3/three-{unique_id}"
-        secret_name_4 = f"/testing4/four-{unique_id}"
+        suffix = random.randint(10000, 99999)
+        secret_name_1 = f"testing1/one-{suffix}"
+        secret_name_2 = f"/testing2/two-{suffix}"
+        secret_name_3 = f"testing3/three-{suffix}"
+        secret_name_4 = f"/testing4/four-{suffix}"
```

Comment:
Does `short_uid()` produce characters that are not allowed in secret names?

Reply:
The problem is that the filter tests are based on characters and a random "a" in the suffix could break the tests. ;)